dag-store.js

```diff
@@ -4,13 +4,10 @@ import uuid from 'node-uuid';
 let nodes = (state = [], action = {}) => {
   switch(action.type) {
```

Comment:
I actually will be using those properties to render back and let people modify the properties they put on the dag.  I did think about doing it that other way (separate map) you are describing but it ends up being a bit more code to maintain and to me this is a bit simpler of a solution to implement.  This small fix in 2 places of the code is a bit simpler than having code to keep the dag and then dag properties separately in sync.  I think it's a nice feature to add and if people want to use it they can or they can use the other approach as well.